src/components/input/range/index.tsx

```diff
@@ -18,31 +17,31 @@ export class SmoothlyInputRange {
 	valueChanged(): void {
 		this.smoothlyInput.emit({ [this.name]: this.value })
 	}
-	inputHandler(e: Event): void {
-		this.isClicked = true
-		e.target instanceof HTMLInputElement &&
-			(this.value = this.step !== "any" ? e.target.valueAsNumber : Math.round(e.target.valueAsNumber * 100) / 100)
+	inputHandler(event: Event): void {
+		event.target instanceof HTMLInputElement &&
+			(this.value =
+				this.step !== "any" ? event.target.valueAsNumber : Math.round(event.target.valueAsNumber * 100) / 100)
 	}
 
 	render() {
 		return (
-			<Host
-				class={{ "is-clicked": this.isClicked }}
-				style={{ "--left-adjustment": `${(this.value / this.max) * 100}%` }}>
-				<label htmlFor={this.name}>
-					<slot />
-				</label>
-				<output class={`output-left`} htmlFor={this.name}>
-					{this.value}
-				</output>
+			<Host style={{ "--left-adjustment": `${(this.value / this.max) * 100}%` }}>
```

Comment:
there seams to be no more references to this css variable in the css. the selector `:host .output-top` no longer hits anything. Remove this and other unused CSS